mmstudio/src/main/java/org/micromanager/data/internal/multipagetiff/OMEMetadata.java

```diff
@@ -391,7 +391,7 @@ private void setOMEDetectorMetadata(Metadata metadata) throws JSONException {
 
       for (int i = 0; i < cameras.size(); i++) {
          String camera = cameras.get(i);
-         String detectorID = MetadataTools.createLSID(camera);
+         String detectorID = MetadataTools.createLSID(camera, i);
```

Comment:
Does this get saved out as "Camera:{i}" or what is the actual string attached to this?

Reply:
I ask because I think this also need to be "Detector:{i}"

Reply:
No, this results in "WhateverNameTheEndUserHaveToThisCamer:{i}". 
I am a bit confused why there are no specific functions to generate these IDs if it is required to provide a given keyword.  This now puts the onus on the poor user of this API to know what keyword is required to generate valid metadata.  Sorry, venting... But it would be nice if you could point me to human understandable documentation.
Happy to change to whatever you think is required.  I'll change according to your suggestions and make a test set available.

Reply:
Same comment as above. Here is the "Dectector ID" Reference: https://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2016-06/ome_xsd.html#DetectorID

Hah no need to apologize. You are talking to a person who had to write an [entire function to clean up old OME XML mistakes](https://github.com/AllenCellModeling/aicsimageio/blob/main/aicsimageio/metadata/utils.py#L59) that were written into our version 3 library -- I understand the gripes of metadata confusion. I am just glad that we started validating our metadata now 😂

Reply:
DataSet1 (using Microscope and cameraName): https://valelab4.ucsf.edu/~nstuurman/images/MMDataSet1/
DataSet2 (using Instrument and Detector): https://valelab4.ucsf.edu/~nstuurman/images/MMDataSet2/
Curious to hear what you find.  And thanks for your efforts!

Reply:
Caught me at just the right moment!

Tested both! Dataset2 (using Instrument and Detector) parses as valid OME and read in just fine in `aicsimageio`.

So I am going to say that does the trick. I also applied the change to a branch of mine as functionality to read the previously generated files: [my cleaning function here](https://github.com/AllenCellModeling/aicsimageio/blob/bugfix/catch-all-errors-in-ome-tiff/aicsimageio/metadata/utils.py#L145)

So both forward and backwards compatibility looks good to me! :tada:

Reply:
Oh woops. Forgot to say, I tried DataSet1 (using Microscope:{i} and Camera:{i}) and it failed to parse as OME. So the Dataset2 route is definitely the way to go.